.github/workflows/dependencies.yml

```
@@ -57,37 +57,8 @@ jobs:
-            ${GITHUB_TOKEN:+-H "Authorization: Bearer ${GITHUB_TOKEN}"} \
-            "https://api.github.com/repos/ImageMagick/ImageMagick/releases/latest" | jq -er '.tag_name')
-          JELLYFIN_FFMPEG_VERSION=$(curl "${CURL_FLAGS[@]}" \
-            ${GITHUB_TOKEN:+-H "Authorization: Bearer ${GITHUB_TOKEN}"} \
-            "https://api.github.com/repos/jellyfin/jellyfin-ffmpeg/releases/latest" | jq -er '.tag_name')
-          # Output as environment variables
-          echo "LIBRAW_VERSION=${LIBRAW_VERSION}" | tee -a "$GITHUB_OUTPUT"
-          echo "LIBHEIF_VERSION=${LIBHEIF_VERSION}" | tee -a "$GITHUB_OUTPUT"
-          echo "IMAGEMAGICK_VERSION=${IMAGEMAGICK_VERSION}" | tee -a "$GITHUB_OUTPUT"
-          echo "JELLYFIN_FFMPEG_VERSION=${JELLYFIN_FFMPEG_VERSION}" | tee -a "$GITHUB_OUTPUT"
+          USER_AGENT: photoview-deps-ci
+        run: ./versions.sh | tee -a $GITHUB_OUTPUT
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:24: Double quote to prevent globbing and word splitting

(shellcheck)